inlong-sort/sort-core/src/test/java/org/apache/inlong/sort/parser/DorisExtractNodeToDorisLoadNodeTest.java

```diff
@@ -0,0 +1,144 @@
+                        new FieldRelation(new FieldInfo("name", new StringFormatInfo()),
+                                new FieldInfo("name", new StringFormatInfo())),
+                        new FieldRelation(new FieldInfo("age", new IntFormatInfo()),
+                                new FieldInfo("age", new IntFormatInfo())),
+                        new FieldRelation(new FieldInfo("price", new DecimalFormatInfo()),
+                                new FieldInfo("price", new DecimalFormatInfo())),
+                        new FieldRelation(new FieldInfo("sale", new DoubleFormatInfo()),
+                                new FieldInfo("sale", new DoubleFormatInfo()))
+                );
+
+        List<FilterFunction> filters = new ArrayList<>();
+        Map<String, String> map = new HashMap<>();
+        return new DorisLoadNode("2", "doris_output", fields, fieldRelations,
+                filters, null, 1, map,
+                "localhost:8030", "root",
+                "000000", "test.test2", null);
+    }
+
+    /**
+     * build node relation
+     *
+     * @param inputs  extract node
+     * @param outputs load node
+     * @return node relation
+     */
+    private NodeRelation buildNodeRelation(List<Node> inputs, List<Node> outputs) {
+        List<String> inputIds = inputs.stream().map(Node::getId).collect(Collectors.toList());
+        List<String> outputIds = outputs.stream().map(Node::getId).collect(Collectors.toList());
+        return new NodeRelation(inputIds, outputIds);
+    }
+
+    /**
+     * Test flink sql task for extract is doris {@link DorisExtractNode} and load is doris {@link DorisLoadNode}
+     */
+    @Test
+    public void testDorisExtractNodeToDorisLoadNodeSqlParse() {
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        env.setParallelism(1);
+        env.enableCheckpointing(10000);
+        env.disableOperatorChaining();
+        EnvironmentSettings settings = EnvironmentSettings
+                .newInstance()
+                .useBlinkPlanner()
+                .inStreamingMode()
+                .build();
+        StreamTableEnvironment tableEnv = StreamTableEnvironment.create(env, settings);
+        Node inputNode = buildDorisExtractNode();
+        Node outputNode = buildDorisLoadNode();
+        StreamInfo streamInfo = new StreamInfo("1", Arrays.asList(inputNode, outputNode),
+                Collections.singletonList(buildNodeRelation(Collections.singletonList(inputNode),
+                        Collections.singletonList(outputNode))));
+        GroupInfo groupInfo = new GroupInfo("1", Collections.singletonList(streamInfo));
+        FlinkSqlParser parser = FlinkSqlParser.getInstance(tableEnv, groupInfo);
+        ParseResult result = parser.parse();
+        try {
+            Assert.assertTrue(result.tryExecute());
+        } catch (Exception e) {
+            e.printStackTrace();

```

Comment:
Please do not use the `e.printStackTrace();` to print all stack trace.

Suggest using the Log framework.

Reply:
ok, i will fix it.